src/shared/components/Dropdown.tsx

```diff
@@ -32,14 +35,22 @@ const SELECTCONTENT_STYLE =
 const SELECTITEM_STYLE =
   'mt-[5px] first:mt-0 data-[highlighted]:bg-black-700 data-[state=checked]:bg-black-700 rounded-md !px-[20px] !py-[6px] text-gray-600 text-sm xl:text-base data-[disabled]:cursor-not-allowed data-[disabled]:opacity-50 data-[highlighted]:text-white data-[state=checked]:text-white';
 
-const Dropdown = ({ initialValue, onChange, placeholder, options }: SelectDropdownProps) => {
+const Dropdown = ({
+  initialValue,
+  onChange,
+  placeholder,
+  options,
+  className,
+  triggerClassName,
+  contentClassName,
+}: SelectDropdownProps) => {
   // 내부 상태는 문자열로 보관 (Radix 규칙)
   const [value, setValue] = useState<string | undefined>(
     initialValue ? String(initialValue.value) : undefined,
   );
 
   return (
-    <div className='w-[355px] md:w-[360px] xl:w-[400px]'>
+    <div className={cn('w-full', className)}>
```

Comment:
w-full만 단독으로 쓰기 보다는 max-w와 조합하는게 더 유연한 사용이 가능합니다. w-full만 사용하면 기본적으로 항상 너비를 꽉 채워 사용자의 커스텀을 강제하지만, w-full 과 각 반응형마다 원래 주려고 했던 너비를 max-w로 주면 부모의 너비가 max-w 값 보다 작아질 때 부모를 꽉 채우도록 변하는 동시에 그보다 크기가 커지면 원래 의도했던 너비로 제한할 수 있습니다. 만약 디자인 변경이 필요하면 className에 max-w-값을 덮어서 커스텀할 수도 있습니다.
```suggestion
    <div className={cn('w-full max-w-[355px] md:w-[360px] xl:max-w-[400px]', className)}>
```